apps/demo/src/components/scrcpy/audio-player.ts

```diff
@@ -0,0 +1,75 @@
+export class AudioPlayer {
+    private _context: AudioContext;
+    private _worklet: AudioWorkletNode | undefined;
+
+    constructor(sampleRate: number) {
+        this._context = new AudioContext({
+            latencyHint: "interactive",
+            sampleRate,
+        });
+    }
+
+    public feed(data: Uint8Array) {
+        this._worklet!.port.postMessage(data.slice());
+    }
+
+    public async start() {
+        // https://github.com/webpack/webpack/issues/11543
+        await this._context.audioWorklet.addModule(
```

Comment:
Try to add a mime override for `.ts` extension and see if `new URL("./audio-player.worker.ts", import.meta.url)` works

Reply:
Not possible. Next.js uses a bundled `mime` package (in `send` package).

MIME also can't be overridden on GitHub Pages.